lib/components/ContextPanel/ContextPanel.module.scss

```diff
@@ -32,30 +32,32 @@ $button-width: 30*$grid-size;
     padding: $gutter-normal;
 }
 
-.content-container {
-    flex-grow: 1;
-}
-
 .title {
     margin: 0;
 }
 
-.header {
+header {
```

Comment:
revert to classname - this will affect all headers in the site, not just the context panel